codice/ddf
# DDF- 2217 cleaned up bower dependencies

#### What does this PR do?
Updates all of the bower files.
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@codice/ui @andrewkfiedler @bcwaters 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@pklinef
@stustison
#### How should this be tested?
build and check that all UIs come up correctly
integration tests + unit tests can all be skipped
#### Any background context you want to provide?
#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Checklist:

 - normalized several different versions of all libraries
 - removed use of github links in bower files
 - updated most libraries to the latest bug fix release
 - updated bower to fix some bugs
